setup.py

```diff
@@ -0,0 +1,19 @@
+from setuptools import setup, find_packages
+
```

Comment:
might be worth just adding a couple of lines in here mentioning this is for building a pip distributable.
As we may end up having a couple of these, as we'll likely need something similar to create an executable